homework/style.css

```diff
@@ -1,3 +1,32 @@
 .alert-error {
-  color: red;
+  color: rgb(128, 50, 50);
+  background-color: rgb(248, 215, 218);
+  padding: 10px 10px;
+}
+* {
+  margin: 0;
+  padding: 0;
+}
+.repos {
+  list-style: none;
+  border: 2px solid rgb(229, 229, 229);
+  padding: 20px 10px;
+}
+
+.header {
+  background-color: rgb(63, 81, 181);
+  border: 2px solid rgb(229, 229, 229);
+  padding: 20px 10px;
+  color: white;
+}
+#root {
```

Comment:
On a wide-screen monitor you use the full width of the screen, which makes your UI look too stretched out. Perhaps set a max width?

![image](https://user-images.githubusercontent.com/2788771/65323831-bb037500-dbaa-11e9-892e-b62bca851bca.png)